nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -58,7 +58,7 @@
       - string:
           name: DISTROS
           description: "A list of distros to build for. Available options are: bionic, centos7, centos8"
```

Comment:
Description should be fixed too

Reply:
my original intent was more to not build it by default and leave it available if needed although I agree I'm not sure this is relevant